native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -399,8 +397,7 @@ JNIEXPORT jobjectArray JNICALL Java_sage_Sage_getRegistrySubkeys(JNIEnv *env, jc
 	
 	RegCloseKey(myKey);
 	env->ReleaseStringUTFChars(key, keyString);
-	return (jobjectArray) env->CallObjectMethod(vec, vecToArrayMeth,
-		env->NewObjectArray(env->CallIntMethod(vec, vecSizeMeth), strClass, NULL));
+	return (jobjectArray) env->CallObjectMethod(vec, vecToArrayMeth, env->NewObjectArray(env->CallIntMethod(vec, vecSizeMeth), strClass, NULL));

```

Comment:
There's a bunch of places in this file where you put multiple lines onto single lines...please revert those because you're going to end up with lines too long for the style that was used in this file.

Reply:
Done, is the style documented somewhere?

Reply:
No, we don't have documented style for this. There's LOTS of style inconsistency in the SageTV codebase, so for new code we can make it look better...but going around changing a bunch of existing code that's not modified for other reasons just makes it harder to track down changes that introduced problems in those same areas.

Reply:
I was having difficulty following the code, so had changed them for readability.
Have reverted them now